sys/include/net/ng_ndp/types.h

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ng_ndp_types    Types for IPv6 neighbor discovery
+ * @ingroup     net_ng_ndp
+ * @brief       IPv6 neighbor discovery message types
+ * @{
+ *
+ * @file
+ * @brief       IPv6 neighbor discovery message type definitions
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NG_NDP_TYPES_H_
+#define NG_NDP_TYPES_H_
+
+#include <inttypes.h>
```

Comment:
> That would be plain evil IMHO :P

I know people who would claim exactly the opposite (and consider putting includes in headers as pure evil). It's a question of philosophy - I've seen both in the context of embedded OS' and think both have benefits and disadvantages.

Since it's common practice to put includes into headers in RIOT, I don't argue to change this, I just try to avoid to put includes in headers that are not required for the header itself, because I think that this make it easier to track them.

Long story short: Replacing `inttypes.h` with `stdint.h` is all I asked for. ;)